src/components/CPU.js

```diff
@@ -47,6 +58,22 @@ class CPU extends PureComponent {
 		);
 	}
 
+	renderCpuSub(commonProps) {
+		// TODO tabs content fill parent
+		return (
+			<div className="CPU__sub">
+				<Tabs onSelect={this.handleSelect}>
+					<TabList>
+						<Tab>Breakpoints</Tab>
+					</TabList>
+					<TabPanel>
+						<BreakpointPanel ppsspp={this.props.ppsspp} {...commonProps}/>
```

Comment:
FWIW, I'm considering switching to a Context strategy for ppsspp/log/etc.:
https://reactjs.org/docs/context.html

Functional components can `useContext()` and class components can have `contextType`, so this'll probably just mean less passing of the same props to every component.

-[Unknown]